oar/core/worksheet.py

```diff
@@ -146,21 +141,48 @@ class TestReport:
     def __init__(self, ws: Worksheet, cs: ConfigStore):
         self._ws = ws
         self._cs = cs
+        self.wm = WorksheetManager(cs)
```

Comment:
```suggestion
```

You don't have to use this. You can use `self._ws` as sheet and `self._ws.spreadsheet` as spreadsheet.